setup.cfg

```diff
@@ -134,6 +134,8 @@ filterwarnings =
     ignore:Distutils was imported before Setuptools
     # Figure tests use agg backend so show doesn't work but we use it in peek
     ignore:Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
+    # We raise this for our HECClient to avoid HTTPS issues with non https urls.
+    ignore:Unverified HTTPS request
```

Comment:
This only came up when I ran the doctests locally. I have not seen this on the CI.